modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/pme_free_switch_test/SingleKeyTxStreamerApplication.java

```diff
@@ -46,11 +43,9 @@
 
             long start = System.currentTimeMillis();
 
-            cache.put(key++, key);
-
-            long finish = System.currentTimeMillis();
+            cache.put(key++ % 100, key); // Cycled update.
 
-            long time = finish - start;
+            long latency = System.currentTimeMillis() - start;
```

Comment:
I don't hesitate to insist, that System.nanoTime() is the only option.

Reply:
Fixed.